include/tvm/tir/stmt.h

```diff
@@ -1339,6 +1339,12 @@ constexpr const char* hand_threaded = "hand_threaded";
  *       if (mask & 2) the write region should be detected.
  */
 constexpr const char* script_parsing_detect_access = "tir.script_parsing_detect_access";
+
+/*!
+ * \brief Mark that the loop should be partitioned.
+ */
+constexpr const char* pragma_loop_partition_hint = "pragma_loop_partition_hint";
```

Comment:
```suggestion
constexpr const char* pragma_loop_partition_hint = "tir.pragma_loop_partition_hint";
```

Reply:
I encounter the problem that the `tir.` prefix seems not compatible with `s[X].pragma(axis, key)`, which add an "pragma_" prefix to key string.